fedora/client/openidbaseclient.py

```diff
@@ -182,7 +183,8 @@ def _authed_get(self, url, params=None, data=None, **kwargs):
         response = self._session.get(url, params=params, data=data, **kwargs)
         return response
 
-    def send_request(self, method, auth=False, verb='POST', **kwargs):
+    def send_request(self, method, auth=False, verb='POST', retries=None,
+                     retry_sleep=0.5, **kwargs):
```

Comment:
Would it make sense to put `retry_sleep` as an attribute of `OpenIdBaseClient`, similarly to `retries`? One could then easily create `Bodhi2Client` and set `Bodhi2Client.retry_sleep` in order to modify the sleep interval.